src/modules/cursor.c

```diff
@@ -105,6 +105,31 @@ static void printCursorXFCE(FFinstance* instance)
     ffStrbufDestroy(&cursorSize);
 }
 
+static void printCursorLXQt(FFinstance* instance)
+{
+    FFstrbuf cursorTheme;
+    ffStrbufInit(&cursorTheme);
+
+    FFstrbuf cursorSize;
+    ffStrbufInit(&cursorSize);
+
+    if(ffParsePropFileConfigValues(instance, "/lxqt/session.conf", 2, (FFpropquery[]) {
+        {"cursor_theme =", &cursorTheme},
+        {"cursor_size =", &cursorSize}
+    })) {
+    }
```

Comment:
I see you copied this over from plasma. `ffParsePropFileConfigValues` return true if the file was found. So i use the if to set the default values when the file exists. This needs to be done in plasma, because default values are not saved. If this isn't the case in LXQt, a simple
```c
ffParsePropFileConfigValues(instance, "/lxqt/session.conf", 2, (FFpropquery[]) {
    {"cursor_theme =", &cursorTheme},
    {"cursor_size =", &cursorSize}
});
```
should be enough.

Also leading slash is not needed in path.